README.md

```diff
@@ -23,20 +20,32 @@ public class TrackScreen implements HasParent<AlbumScreen> {
 }
 ```
 
-The `HasParent` interface is used to support the *up* notion used in Android.
-
 ## Backstack
 
-The backstack is the history of screens, with the head being the current or last-most screen.
+The backstack is the history of paths, with the head being the current path.
 
 ## Flow
 
-The flow holds the current truth about your application, the history of screens. It can be told to transition to another screen by simply instantiating the screen you want to go to.
+The Flow holds the current backstack and offers navigation.
```

Comment:
I agree, but going to go ahead and land this and do a more focused pass on the README as part of the next release. Will clean it up there for sure.